src/components/calcite-label/calcite-label.tsx

```diff
@@ -34,6 +34,9 @@ export class CalciteLabel {
   @Prop({ mutable: true, reflect: true }) layout: "inline" | "inline-space-between" | "default" =
     "default";
 
+  /** Turn off margin around the label */
+  @Prop() noMargin?: boolean;
```

Comment:
maybe `disable-margin`, `frameless` or something less prop based and more based on functionality/intention?

Reply:
If we change this, we should probably also change `noPadding` on `calcite-modal`

Reply:
Yes let's just make sure Calcite-modal and Calcite-loader "no-padding" end up matching. Disable-margin/padding?

Reply:
I agree. I never like it to be that specific.
TBH, I was thinking of just tightening up the bottom margin on label...at least making it scale dependent. That might help the case that Adelheid is dealing with.

Reply:
That's added here: https://github.com/Esri/calcite-components/pull/914/files#diff-ad18c8f8961f619818ad69c1d1c15b17R9 , scale-dependent bottom margin. But I think we'd still need a "disable-margin" option

Reply:
It does sound a bit too specific about what it is doing under the hood. Something more general would be nice.

Reply:
Maybe `compact`? Could be a boolean that means "take extra space away".

Reply:
Compact kind of seems a bit close to scale=s... we already have no-padding, I agree disable-margin/padding is better than that. 

`disable-spacing` could be used on all three components.

 Although that could insinuate it's disabling internal spacing...